apps/dashboard/src/contract-ui/tabs/embed/components/embed-setup.tsx

```diff
@@ -159,6 +159,58 @@ const buildIframeSrc = (
   return url.toString();
 };
 
+// taken from @thirdweb-dev /chains
+// @internal
+function minimizeChain(
+  chain: ChainMetadata,
+): Pick<
```

Comment:
This Pick is so long it's probably worth it to just redefine the whole type

Reply:
I'd rather just delete all of this when we remove embed support :) - for now it's fine to just yoink it out of the old package